### DISCUSSION TITLE
Landing Page UI Screenshots

### DISCUSSION THREAD

**Discussion by stolinski** [coollabsio/coolify#1676] [Feature Requests]
This isn't a suggestion for the application itself, but I was just looking at considering this project for myself however, one thing I noticed that might be important to other people looking is that I didn't see any screenshots of the application at least on mobile. Just something to consider that I think people will want to see the UI much more in the documentation.

**Comment by andrasbacsai** [comment]
Thanks for the suggestion! More and more people are asking for this. Initially, I thought I would wait until I redesign a few views before taking any screenshots, but I can change the screenshots anytime, so I will do it now!

**Comment by andrasbacsai** [comment]
Done, added a few screenshots on the [landing page](https://coolify.io).
